source/overloading.tex

```diff
@@ -1548,7 +1548,8 @@
 of the \grammarterm{deduction-guide}.
 \end{itemize}
 \end{itemize}
-In addition, if \tcode{C} satisfies the conditions for an aggregate class
+In addition, if \tcode{C} satisfies the conditions for an
+aggregate class\iref{dcl.init.aggr}
```

Comment:
This paragraph doesn't follow the semantic line breaks convention, but since you're inserting a newline here anyway, I think the newline should be before "an" so that it is grouped with "aggregate class".

Reply:
It was a novice attempt at introducing a semantic line break.  (I've always used either regular filling or line-per-sentence.)  I'll fix it in the next push.